### DISCUSSION TITLE
用油猴启用脚本后，功能面板和b站顶栏出现乱码

### DISCUSSION THREAD

**Discussion by 12208583** [the1812/Bilibili-Evolved#3206] [提问 / Q&A]
浏览器：Firefox 98.0.2   脚本版本：1246.26

![屏幕截图 2022-04-01 192610](https://user-images.githubusercontent.com/68137087/161255645-94b7c869-c23f-48f9-877a-9732bca80881.png)

**Comment by 12208583** [comment]
求解惑，谢谢

**Comment by Geeyun-JY3** [comment]
你这是 v1 吧，该升 v2 了。记得认真阅读 README 哦。

**Reply by 12208583** [reply]
是的，可升级最新一个版本后（2.1.7）面板上乱码依旧存在。查看js源码，里面似乎也有乱码
![屏幕截图 2022-04-02 093212](https://user-images.githubusercontent.com/68137087/161360484-d4afc6c6-4464-4f6a-81a1-affb896b3697.png)
![屏幕截图 2022-04-02 093359](https://user-images.githubusercontent.com/68137087/161360486-71687ffe-bac3-45cf-8003-4be33d5e8cc7.png)

**Reply by Geeyun-JY3** [reply]
如果排查过不是其他脚本或扩展的问题，完全删除脚本后再重装也是这样，建议贴报错。

**Reply by 12208583** [reply]
好的，谢谢
